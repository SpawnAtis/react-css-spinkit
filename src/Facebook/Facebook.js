import React from 'react'
import styles from './styles.css'

const Facebook = () => {
  return (
    <>
      <style type="text/css">{`${styles}`}</style>
      <div className="lds-facebook">
        <div />
        <div />
        <div />
      </div>
    </>
  )
}

export default Facebook
