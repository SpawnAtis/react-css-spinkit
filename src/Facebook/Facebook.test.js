import React from 'react'
import {render} from 'react-testing-library'
import Facebook from './Facebook'

describe('Facebook component', () => {
  it('Should render Facebook without problems', () => {
    const {asFragment} = render(<Facebook />)
    expect(asFragment()).toMatchSnapshot()
  })
})
