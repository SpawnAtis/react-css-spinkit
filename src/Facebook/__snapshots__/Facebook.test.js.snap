// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Facebook component Should render Facebook without problems 1`] = `
<DocumentFragment>
  <style
    type="text/css"
  >
    [object Object]
  </style>
  <div
    class="lds-facebook"
  >
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
